plugins/MumblePlugin.h

```diff
@@ -1448,6 +1449,24 @@ struct MUMBLE_API_STRUCT_NAME {
 	 */
 	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *isLocalUserDeafened)(mumble_plugin_id_t callerID, bool *deafened);
 
+#	if SELECTED_API_VERSION >= MUMBLE_PLUGIN_VERSION_CHECK(1, 3, 0)
+
+	/**
+	 *
+	 * Checks whether the linkedChannelID is linked to channelID.
+	 *
+	 * @param callerID The ID of the plugin calling this function
+	 * @param channelID The ID of the channel
+	 * @param linkedChannelID The ID of the linked channel
+	 * @param[out] linked A pointer to where the result of the check
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *isChannelLinkedTo)(mumble_plugin_id_t callerID,
+																		mumble_channelid_t channelID,
+																		mumble_channelid_t linkedChannelID,
+																		bool *linked);
+
+#	endif
```

Comment:
Looking at other functions in this block of API functions it seems like they wildly mix is* and get* functions. Therefore, I don't see a need to put this function up here and `getLinkedChannels` further down. I would put both functions together where `getLinkedChannels` currently is - that way they can also share the preprocessor guard w.r.t. API version